### DISCUSSION TITLE
Luigi's Mansion 3

### DISCUSSION THREAD

**Discussion by Rimurudt** [skyline-emu/skyline#419] [Game Requests]
No Luigi's mansion 3 k

**Comment by iniMerchant64** [comment]
No. Maybe in 2 years

**Reply by Rimurudt** [reply]
2 year's why?

**Reply by iniMerchant64** [reply]
From what i can remember ryujinx devs mentioned it to be one of the hardest games to emulate properly. Its rather painful to get it working so id assume it will take way longer than anything else

**Comment by PixelyIon** [comment]
This is up there on the list being a 1st party Nintendo game. I also already have it dumped so that in itself boosts it up a few places.

**Comment by Rimurudt** [comment]
How about cuphead? Is that easier to emulate?

**Reply by iniMerchant64** [reply]
Its a unity game, another game request

**Reply by masoom1786raza** [reply]
Resident evil series it's great to play zombie survival games on our smartphone. Our device is capable to play resident evil 4, my device cpu Is snapdragon 636 cortex A73 it is very playable I tried dolphin wii/gamecube to play it it works great. It gaves me 30fps on a low end device. Add features like overclocking ee cycle skip, underclocking ee cycle it gives a huge performance inhancement.
